js102/index.html

```diff
@@ -318,8 +390,37 @@
 				</section>
 				<!-- /Nested Objects -->
 
+				<!-- Functions Intro -->
+				<section>
+					<h2>So, what about Functions?</h2>
+					<ul>
+						<li>first-class object because they have properties and methods</li>
```

Comment:
Can you explain what first-class means? This might be a bit jargon-y if we don't break it down